recipe/meta.yaml

```diff
@@ -54,23 +55,56 @@ outputs:
         - perl  # [osx]
         - autoconf  # [osx]
         - automake  # [osx]
-        - libtool   # [unix]
+        - binutils  # [linux]
+        - libevent
+        - libhwloc
+        - libmagic
+        - libnl  # [linux]
+        - libnuma  # [linux]
+        - libtool  # [unix]
+        - libudev1  # [linux]
         - make  # [unix]
+        - numactl  # [linux]
+        - pkg-config 
+        - rdma-core  # [linux]
+        - ucc  # [linux and not osx and not ppc64le]
+        - ucx  # [linux and not osx]
+        - zlib
+          
       host:
         #- openpmix
         #- prrte
-        - libhwloc
+        - binutils  # [linux]
         - libevent
+        - libhwloc
+        - libmagic
         - libnl  # [linux]
+        - libnuma  # [linux]
+        - libudev1  # [linux]
+        - numactl  # [linux]
+        - pkg-config  # [linux]
+        - rdma-core  # [linux]
+        - ucc  # [linux and not osx and not ppc64le]
+        - ucx  # [linux and not osx]
         - zlib
-        - ucx  # [linux]
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler != "None"]
       run:
         - mpi 1.0 openmpi
         #- openpmix
         #- prrte
+        - libevent
+        - libhwloc
+        - libnl  # [linux]
+        - libnuma  # [linux]
+        - libudev1  # [linux]
+        - numactl  # [linux]
+        - rdma-core  # [linux]
+        - ucc  # [linux and not osx and not ppc64le]
+        - ucx  # [linux and not osx]
```

Comment:
ucx is specfically not a runtime dependency, and I _imagine_ ucc should be treated the same. Try removing _all_ of these new runtime dependencies, leaving only the `run_constrained` addition.

Reply:
OK